src/components/notifications/GtxMessaging.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div class="mb-3">
+        <label for="gtxmessaging-api-key" class="form-label">{{ $t("gtxMessagingApiKey") }}</label>
+        <HiddenInput id="gtxmessaging-api-key" v-model="$parent.notification.gtxMessagingApiKey" :required="true"></HiddenInput>
+    </div>
+    <div class="mb-3">
```

Comment:
Please add a helptext and placeholders for both `From` and `To` better explaining how they can be used
![image](https://github.com/louislam/uptime-kuma/assets/26258709/5fc20ad9-e281-43bc-b6c5-a9d0b85ded38)